### ISSUE TITLE
Enhance Figma documentation on website.

### ISSUE THREAD

**Issue by edonehoo** [patternfly/patternfly-org#3865]
Follow-up once #3863 is done in order to expand on Figma documentation to add more details. Clean up (and consider moving) Sketch documentation at the same time.

**Comment by edonehoo** [comment]
Closing this in favor of #3889 which will include these updates
